### PR TITLE
fix: Attachments on intent request change to store the data in the request object

### PR DESCRIPTION
**Pull request by OpenDog**
The FB channel will use it (for now) in core.

### CODE REVIEW

File: packages/stentor-models/src/Request/IntentRequest.ts [hunk 1]
```diff
@@ -117,7 +128,11 @@ export interface IntentRequest extends BaseRequest {
...
     data?: Data;
     /**
-     * A unique request provided by a question answering system.  
+     * A unique request provided by a question answering system.
      */
     knowledgeAnswer?: KnowledgeAnswer;
+    /**
+     * Uploads from the request
+     */
+    attachmentAnswer?: AttachmentAnswer[];
```
Comment (michaelmyers) [inline packages/stentor-models/src/Request/IntentRequest.ts]: How about just `attachment`?

Comment (OpenDog) [reply packages/stentor-models/src/Request/IntentRequest.ts]: Done.

File: packages/stentor-models/src/Request/IntentRequest.ts [hunk 2]
```diff
@@ -25,6 +25,17 @@ export interface KnowledgeAnswer {
     matchConfidence: number;
 }
 
+export interface AttachmentAnswer {
```
Comment (michaelmyers) [inline packages/stentor-models/src/Request/IntentRequest.ts]: How about `RequestAttachment`?

File: packages/stentor-models/src/Request/Types.ts [hunk 3]
```diff
@@ -30,3 +30,4 @@ export type InputUnknownID = "InputUnknown";
 export type OptionSelectID = "OptionSelect";
 export type SignInID = "SignIn";
 export type KnowledgeAnswerID = "KnowledgeAnswer";
+export type AttachmentAnswerID = "AttachmentAnswer";
```
Comment (michaelmyers) [inline packages/stentor-models/src/Request/Types.ts]:
```suggestion
export type AttachmentRequestID = "AttachmentRequest";
```

File: packages/stentor-request/src/Constants.ts [hunk 4]
```diff
@@ -36,6 +37,7 @@ export const PERMISSION_GRANT_ID: PermissionGrantID = "PermissionGrant";
 export const OPTION_SELECT_ID: OptionSelectID = "OptionSelect";
 export const SIGN_IN_ID: SignInID = "SignIn";
 export const KNOWLEDGE_ANSWER_ID: KnowledgeAnswerID = "KnowledgeAnswer";
+export const ATTACHMENT_ANSWER_ID: AttachmentAnswerID = "AttachmentAnswer";
```
Comment (michaelmyers) [inline packages/stentor-request/src/Constants.ts]:
```suggestion
export const ATTACHMENT_REQUEST_ID: AttachmentRequestID = "AttachmentRequest";
```

**Pull request merged**